src/context/providers/HubbardContextProvider.js

```diff
@@ -0,0 +1,83 @@
+import { Application } from "@exabyte-io/ade.js";
+import {
+    ApplicationContextMixin,
+    JSONSchemaFormDataProvider,
+    MaterialContextMixin,
+} from "@exabyte-io/code.js/dist/context";
+import { Made } from "@exabyte-io/made.js";
+import { mix } from "mixwith";
+
+const defaultHubbardConfig = {
+    atomicSpecies: "",
+    atomicOrbital: "2p",
+    hubbardUValue: 0.01,
+};
+
+export class HubbardContextProvider extends mix(JSONSchemaFormDataProvider).with(
+    ApplicationContextMixin,
+    MaterialContextMixin,
+) {
+    static Application = Application;
+
+    static Material = Made.Material;
+
+    constructor(config) {
+        super(config);
+        this.uniqueElements = this.material.Basis.uniqueElements;
```

Comment:
to be safe, we should initialize as empty list if there is an issue with the material, for instance:
```javascript
this.uniqueElements = this.material?.Basis?.uniqueElements || [];
```